src/google_map.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies, react/forbid-prop-types, react/no-find-dom-node, no-console, no-undef */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import ReactDOM from 'react-dom';
+import ReactDOM, { flushSync } from 'react-dom';
```

Comment:
is `flushSync` a new API that requires react `v18`?

If yes, you should update the react version to force require v18?

https://github.com/theplant/google-map-react/blob/master/package.json#L37
https://github.com/theplant/google-map-react/blob/master/package.json#L59

Reply:
> is flushSync a new API that requires react v18?

No, it's already there on v16 https://github.com/facebook/react/blob/v16.1.0/packages/react-dom/src/client/ReactDOM.js#L943.
So it should still complicate with react v16 and v17

Reply:
Good to know. Thank you @lizhengchao 

Then this change is a patch update.

Reply:
> Then this change is a patch update.

Yes, it should be a patch update.

Reply:
that's great.